src/vaults/vaults.service.ts

```diff
@@ -215,17 +244,18 @@ export class VaultsService {
     value: number,
     pendingValue: number,
     lastHarvested: number,
-    apy = false,
+    compoundingValue = 0,
   ): number {
     if (lastHarvested === 0 || value === 0 || pendingValue === 0) {
       return 0;
     }
     const duration = Date.now() / 1000 - lastHarvested;
-    const apr = (pendingValue / value) * (ONE_YEAR_SECONDS / duration);
-    if (!apy) {
-      return apr * 100;
+    const apr = (pendingValue / value) * (ONE_YEAR_SECONDS / duration) * 100;
```

Comment:
this compounding value is then passed through here to either use to calculate apy or to treat just as apr
in other words, we would use that amount that gets compounded and calculate the apy instead